packages/lesswrong/server/utils/bundleUtils.ts

```diff
@@ -44,3 +44,15 @@ export const getClientBundle = () => {
   
   return clientBundle;
 }
+
+let serverBundleHash: string|null = null;
+export const getServerBundleHash = (): string => {
+  if (!serverBundleHash) {
+    const serverBundlePath = path.join(__dirname, "../../server/js/serverBundle.js");
```

Comment:
Do we have a way to confirm that this path is correct on the remote server (and not just here)?  I think I remember running into some issues with path construction because of mismatches like that in the past

Reply:
I think it should be ok because we do the same thing earlier in the file, for the client bundle.